cs-jclouds/src/main/java/io/cloudslang/content/jclouds/factory/helpers/ImageHelper.java

```diff
@@ -0,0 +1,183 @@
+package io.cloudslang.content.jclouds.factory.helpers;
+
+import io.cloudslang.content.jclouds.entities.constants.Constants;
+import io.cloudslang.content.jclouds.entities.inputs.InputsWrapper;
+import io.cloudslang.content.jclouds.utils.InputsUtil;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Set;
+
+import static io.cloudslang.content.jclouds.entities.constants.Constants.AwsParams.ATTRIBUTE;
+import static io.cloudslang.content.jclouds.entities.constants.Constants.AwsParams.EXECUTABLE_BY;
+import static io.cloudslang.content.jclouds.entities.constants.Constants.AwsParams.FILTER_NAME;
+import static io.cloudslang.content.jclouds.entities.constants.Constants.AwsParams.FILTER_VALUE;
+import static io.cloudslang.content.jclouds.entities.constants.Constants.AwsParams.INSTANCE_ID;
+import static io.cloudslang.content.jclouds.entities.constants.Constants.AwsParams.NO_REBOOT;
+import static io.cloudslang.content.jclouds.entities.constants.Constants.AwsParams.OPERATION_TYPE;
+import static io.cloudslang.content.jclouds.entities.constants.Constants.AwsParams.USER_GROUP;
+import static io.cloudslang.content.jclouds.entities.constants.Constants.AwsParams.USER_ID;
+
+
+/**
+ * Created by Tirla Alin.
+ * Date: 30/9/2016.
+ */
+public class ImageHelper {
```

Comment:
Currently all other structures (network, instances etc) have "helpers". Should i rename it?